NextCenturyCorporation/itm-evaluation-server
# ITM-86: Add gradle config for swagger generation

Treat this as a work in progress, I haven't removed the existing models yet because its easier to verify that things are being generated correctly while git can still diff.

To test:
Manually delete the swagger_server/models folder
refresh your IDE or do a "git status" on the command line to see that git see the files as missing

run "./gradlew" to run the swagger generation

refresh your IDE or do a "git status" again. If all the models are generated correctly there should be no changes from gits perspective. The only file that is different in the __init__.py but its only unnecessary comments that are changed.

Once this is verified I'll do the clean up of removing the models from git and adding the git ignore.
